scroll-animations-1/Overview.bs

```diff
@@ -736,67 +736,79 @@ spec: cssom-view-1; type: dfn;
 
 	A set of animation keyframes can be attached
 	in reference to [=named timeline ranges=]
-	by adjusting the duration of the animation.
+	by restricting the to the specified timeline range.
+	Delays ('animation-start-delay'/'animation-end-delay')
+	are set within this restricted range.
+	If a duration ('animation-duration') is given,
+	it measures from the range-restricted start,
+	and further restricts the range-restricted end.
```

Comment:
It is just used instead rather than replacing it, isn't it? E.g. similar to how width is just used instead of right for absolute positioning regardless of whether that further restricts or expands the width: https://jsbin.com/bekiceg/edit?html,css,output

Reply:
I think it would make more sense to take the more restrictive one, since if you set a shorter animation-range-end and a longer duration, it'd seem weird to ignore the animation-range-end?

Reply:
I'm confused by the interaction with duration here. Ranges relate to `ViewTimeine`s,  and these don't have a specified duration (unless we specify interaction with `animation-iteration-count`) but rather a duration of `auto` or `100%`, right?
So unless there's some sort of `animation-iteration-count: infinite/auto` i'd expect `animation-duration` to be ignored and be calculated from ranges and delays.

Reply:
@ydaniv That's another approach we could take: if animation-range-end has a valid non-normal value, ignore the duration. (It seemed the discussion in an earlier issue wanted them to both apply somehow.)

Reply:
Yes, I thought that's the main use-case, so you're saying range-end could be `normal` and duration in <length> can determine the actual duration, staring from the range-start? 
SGTM, I'd also prefer to respect the range-end and clamp to that

Reply:
So taking the shorter of `<range-end> - <range-start>` and `<duration>` still isn't really honoring the request:
- If the range is shorter, you're not using the specified duration
- If the duration is shorter, the animation doesn't end at range-end.

If we don't need to follow the precedent for box positioning, I'd suggest that when a start and end range are both not `normal` we use that instead of the duration.

Reply:
+1

Reply:
OK, I'll open a new issue about this. -> https://github.com/w3c/csswg-drafts/issues/8405